documentation/minimal_viable_product_description.md

```diff
@@ -0,0 +1,125 @@
+# **MVP Pipeline (Target: Start of November)**
+
+## **Primary Goal**  
+The new pipeline should generate predictions that are consistent with the old pipeline’s performance while incorporating HydraNet PGM-level predictions. Although some model implementations will differ, the overall predictive performance is expected to be as good as, or better than, the old pipeline.
+
+---
+
+## **MVP Requirements and Deliverables**
+
+### **1. PGM Predictions from Legacy Models**  
+Produce PGM-level predictions from the majority of models implemented in the old pipeline. The models to be included should be referenced from the model [catalogs](https://github.com/prio-data/views_pipeline/tree/main/documentation/catalogs), which track the transition to the new pipeline. Where applicable, unmodified DARTS models should be considered if their classification performance meets the project’s needs. Adaptation efforts for any individual model should not exceed three working days. Any models that cannot be adapted within this timeframe should be deprioritized for now. Unmaintainable or deprecated models will not be included in the MVP.
+
```

Comment:
It is, in my opinion, essential that the old stepshifter not be used by any of the models in the new pipeline, since it creates so many maintainability and dependency problems. I believe it has been eliminated from all but a few models. However, this creates the problem of implementing the hurdle models, which cannot be done using either DARTS or scikit-forecast as far as I am aware, although Xiaolong and Simon are working on a workaround.